_data/events_gdcr2018/Seattle-codecrafters.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "GDCR 2018 - Seattle Code Crafters",
+  "url": "https://www.meetup.com/seattle-software-craftsmanship/events/254972860/",
+  "moderators": [
+    "@katanation",
+    "@paigeisxp"
+  ],
+  "location": {
+    "city": "Lynnwood",
+    "country": "United States of America",
+    "coordinates": {
+      "latitude": 47.845791,
+      "longitude": -122.293084
+    },
+    "utcOffset": -7,
+    "timezone": "America/Los_Angeles"
+  }
+}
```

Comment:
Could we remove the line ending character here?

Reply:
I am not seeing this in my repository.  Could be that VSCode is not showing it??

Reply:
I am only talking about the red mark(s) of [line 18](https://github.com/coderetreat/coderetreat.github.io/pull/359/files#diff-e729007269a07255378aa09edbc3111dR18). What I mean, is that I expected it to look like [this line 18](https://github.com/coderetreat/coderetreat.github.io/pull/358/files#diff-5ec46ec8d019dec7df83b51d811ed4caR18) (, so, without any red marks at all).

However, given the following facts:

- we have already merged [this one](https://github.com/coderetreat/coderetreat.github.io/pull/357) and it didn't break anything
- we have no test case for newline policies for git, linux and windows, there is no actual proof that this will go wrong.

So, for parking this for now and will investigate it further with #360